Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
\ No newline at end of file
+This is how you change a file.
```

Comment:
Well, maybe not.

---
In reply to: [56032577](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r56032577) [](ancestors = 56032577)